index.js

```diff
@@ -1,3 +1,5 @@
+import { createCharacterCard } from "./components/card/card.js";
```

Comment:
As this is going to be a default export (see my comment above), you will need to adjust this import:

```suggestion
import createCharacterCard from "./components/card/card.js";
```